devenv.nix

```diff
@@ -56,83 +56,53 @@ let
   };
 
   entryScript = pkgs.writeScript "entryScript" ''
-    PATH="${lib.makeBinPath [ pkgs.coreutils ]}:$PATH"
+    set -euo pipefail
 
     while ! $DEVENV_PROFILE/bin/mysqladmin ping --silent; do
-      sleep 1
+      ${pkgs.coreutils}/bin/sleep 1
     done
 
     ${scriptUpdateConfig}
 
     echo -e "Startup completed"
 
-    sleep infinity
+    ${pkgs.coreutils}/bin/sleep infinity
   '';
 
   systemConfigEntries = lib.mapAttrsToList (name: value: { inherit name value; }) cfg.systemConfig;
 
   scriptUpdateConfig = pkgs.writeScript "scriptUpdateConfig" ''
+    VENDOR=${config.env.DEVENV_ROOT}/vendor/autoload.php
+    CONSOLE=${config.env.DEVENV_ROOT}/bin/console
+
+    echo "Updating system config"
+
+    if [ ! -f "$VENDOR" ] || [ ! -f "$CONSOLE" ];
+    then
+      echo "Vendor folder or console command not found. Please run composer install."
+      exit 1
+    fi
+
     # additional config
     ${lib.concatMapStrings ({ name, value }: ''
-      ${updateConfig} ${name} "${lib.escapeShellArg value}"
+      $CONSOLE system:config:set ${name} ${value} || exit 1
```

Comment:
There should be a specific message?

Reply:
Errors are always displayed in devenv :)